nilrt_snac/_Configs/_NtpConfig.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import subprocess
+
+from nilrt_snac._Configs._BaseConfig import _BaseConfig
+from nilrt_snac._Configs._ConfigFile import _ConfigFile
+
+from nilrt_snac import logger
+from nilrt_snac.OpkgHelper import opkg_helper
+
+
+class _NtpConfig(_BaseConfig):
+    def __init__(self):
+        self._config_file = _ConfigFile("/etc/ntp.conf")
```

Comment:
A note: by loading the config files in the Config.`__init__()`s, it means that any two configuration classes that modify the same config file will only end up writing the latest class' changes. Because they will all load the original (pre-run) version of the config file. You might move this logic into the `configure()` and `verify()` methods instead, and `save()` at the and of each method execution.